float32 x_gyro
float32 y_gyro
float32 z_gyro
